### ISSUE TITLE
Can't get Samurai Showdown (Neo Geo) to play on SNES Classic

### ISSUE THREAD

**Issue by ronin322** [ClusterM/retroarch-clover#117]
I've changed ".../zip " to ".../fba ", and included the Neo Geo bios (neogeo.zip) in its game folder, but all I get is a black screen before it returns to game selection menu. The module was definitely installed. What am I missing? This is my first Neo Geo game test so there are no titles for Neo Geo working on my console yet.

**Comment by Moryggy** [comment]
Have you tried to launch RetroArch itself and launch it from there to test if it loads at all or not?

Do you have neogeo bios in the same folder? I'm not sure about this, but I'm think it must be there.

I could not run Metal Slug 1, but I can run from RetroArch Metal Slug 2 and Neo Turf Masters. I'm still testing things because I think I could know the root cause but I just can test during a short time window every evening.

**Comment by ronin322** [comment]
No, I haven’t tried launching via RetroArch. Will do though! The neogeo.zip bios is in the game folder. Will report back after attempt. Thanks for the suggestion.

**Comment by ronin322** [comment]
When I load the rom from within RetroArch, I end up with “No items.”
